solr/bin/solr

```diff
@@ -1210,12 +1101,8 @@ function start_solr() {
   fi
 
 
-  # If SSL-related system props are set, add them to SCRIPT_SOLR_OPTS
-  if [ "$SOLR_SSL_ENABLED" == "true" ]; then
-    # If using SSL and solr.jetty.https.port not set explicitly, use the solr.port.listen
-    SSL_PORT_PROP="-Dsolr.jetty.https.port=$SOLR_PORT_LISTEN"
-    SCRIPT_SOLR_OPTS+=($SOLR_SSL_OPTS "$SSL_PORT_PROP")
-  fi
+  # SIP-6: SSL configuration is now handled by ServerConfiguration and ConnectorFactory in Java
+  # No need to pass SSL properties explicitly - they're read from environment variables
```

Comment:
I suppose eventually we can remove these comments, but nice to see now in draft mode!

Reply:
Sure, this is just POC filler. 

If we go this direction, it will open up for porting most of the start script to Java, except the JVM level props like heap, gc etc.